packages-next/fields/src/types/relationship/views/cards/index.tsx

```diff
@@ -123,125 +133,150 @@ export function Cards({
-                      </FieldContainer>
-                    );
-                  }
-                  itemForField[graphqlField] = fieldGetter.data;
-                }
+      <Stack
+        as="ul"
+        gap="xlarge"
+        css={{
+          padding: 0,
+          marginBottom: 0,
+          li: {
+            listStyle: 'none',
+          },
+        }}
+      >
+        {[...value.currentIds].map((id, index) => {
+          const itemGetter = items[id];
 
-                return (
-                  <field.views.CardValue
-                    key={fieldPath}
-                    field={field.controller}
-                    item={itemForField}
-                  />
-                );
-              })}
-            </Stack>
-            <Stack across gap="small" marginTop="xlarge">
-              {field.display.inlineEdit && onChange !== undefined && (
-                <Button
-                  size="small"
-                  disabled={onChange === undefined}
-                  onClick={() => {
-                    onChange({
-                      ...value,
-                      itemsBeingEdited: new Set([...value.itemsBeingEdited, id]),
-                    });
-                  }}
-                  tone="active"
-                >
-                  Edit
-                </Button>
-              )}
-              {field.display.removeMode === 'disconnect' && onChange !== undefined && (
-                <Tooltip content="This item will not be deleted. It will only be removed from this field.">
-                  {props => (
-                    <Button
-                      size="small"
-                      disabled={onChange === undefined}
-                      onClick={() => {
-                        const currentIds = new Set(value.currentIds);
-                        currentIds.delete(id);
-                        onChange({
-                          ...value,
-                          currentIds,
-                        });
-                      }}
-                      {...props}
-                      tone="negative"
-                    >
-                      Remove
-                    </Button>
-                  )}
-                </Tooltip>
-              )}
-              {field.display.linkToItem && (
-                <Button
-                  size="small"
-                  weight="link"
-                  tone="active"
-                  css={{ textDecoration: 'none' }}
-                  as={Link}
-                  href={`/${foreignList.path}/${id}`}
-                >
-                  View {foreignList.singular} details
-                </Button>
-              )}
-            </Stack>
-          </CardContainer>
-        );
-      })}
+          return value.itemsBeingEdited.has(id) && onChange !== undefined ? (
+            <CardContainer mode="edit" key={id}>
+              <VisuallyHidden as="h2">{`${field.label} ${index + 1}: Edit mode`}</VisuallyHidden>
```

Comment:
Adding a visually hidden heading to demarcate and differentiate list item content to screen readers. 
specifically we're denoting that an element has entered 'edit' mode here.